data/remnant/remnant news.txt

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2020 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+phrase "00-07"
+	word
+		"00"
+		"01"
+		"03"
+		"04"
+		"05"
+		"06"
+		"07"
```

Comment:
```suggestion
	word
		"0"
	word
		"0"
		"1"
		"3"
		"4"
		"5"
		"6"
		"7"
```

Reply:
I intentionally used double digits for stylistic purposes.